Pure_Parallels/update_parallel_wcs.py

```diff
@@ -0,0 +1,155 @@
+import os, sys
+import shutil
+import glob
+from astropy.io import ascii
+import astropy.io.fits as pyfits
+import numpy as np
+
+def log_comment(LOGFILE, comment, verbose=False, show_date=False, mode='a'):
+    """
+    Log a message to a file, optionally including a date tag
+    """
+    import time
```

Comment:
To adhere to the PEP8 standards all imports should be at the top of the file.

Reply:
done